templates/docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM UBUNTU_DUMMY_VERSION
```

Comment:
This file should be better placed somewhere in configurations and not in the templates folder. It is not really a template. Or do you have other idea/opinion?